packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/MrwSerializationTypeDefinition.cs

```diff
@@ -660,8 +722,9 @@ private MethodBodyStatement CallBaseJsonModelWriteCore()
                 : MethodBodyStatement.Empty;
         }
 
-        private MethodBodyStatement GetPropertyInitializers(IReadOnlyList<ParameterProvider> parameters)
+        private MethodBodyStatement GetPropertyInitializers(IEnumerable<ParameterProvider> parameters)
```

Comment:
we might be able to remove the param here and loop on my properties

Reply:
fixed.